modules/workflow/src/main/resources/processes/migrate.bpmn20.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
does this file really need 'bpmn20' to be appended to its name? i would prefer to have it without it.

Reply:
Make sure that the file ends with .bpmn20.xml or .bpmn, since otherwise the engine won’t pick up this file for deployment.
![image](https://github.com/eclipse-xpanse/xpanse/assets/122504203/e26f17af-7f36-4779-bbab-d3a9ae662cd7)